lib/sisimai/bite/email/mailfoundry.rb

```diff
@@ -0,0 +1,133 @@
+module Sisimai::Bite::Email
+  module MailFoundry
+    # Sisimai::Bite::Email::MailFoundry parses a bounce email which created by
+    # MailFoundry. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/MailFoundry.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :subject  => %r/\AMessage delivery has failed\z/,
+        :received => %r/[(]MAILFOUNDRY[)] id /,
+      }
+      Re1 = {
+        :begin  => %r/\AThis is a MIME encoded message\z/,
+        :error  => %r/\ADelivery failed for the following reason:\z/,
+        :rfc822 => %r|\AContent-Type: message/rfc822\z|,
+        :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'MailFoundry'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
+      def headerlist;  return []; end
+      def pattern;     return Re0; end
+
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->